lib/model/image_sscd.py

```diff
@@ -0,0 +1,58 @@
+from typing import Dict
+import io
+
+# from lib.model.generic_image import GenericImageModel
+from lib.model.model import Model
+from lib import schemas
+from torchvision import transforms
+import torch
+from lib.logger import logger
+import numpy as np
+from PIL import Image
+import urllib.request
+
+class Model(Model):
+    def __init__(self):
+        super().__init__()
+        #FIXME: Load from a Meedan S3 bucket
+        try:
+            self.model = torch.jit.load("sscd_disc_mixup.torchscript.pt")
+        except:
+            logger.info("Downloading SSCD model...")
+            m=urllib.request.urlopen("https://dl.fbaipublicfiles.com/sscd-copy-detection/sscd_disc_mixup.torchscript.pt").read()
```

Comment:
Please make all these strings constants and/or environment variables. Also - @computermacgyver do we want to host this in our own infrastrcture?

Reply:
I'm OK with keeping the URL pointing to fbaipublicfiles.com for now. I have a backup in our own S3 bucket in case we need it at some point.

Reply:
Where did we land with running this within the existing image model? Do we still need to refactor for that?